ext/telemetry/lib.rs

```diff
@@ -678,12 +776,25 @@ fn op_otel_log(
     );
   }
 
-  log_processor.emit(
+  logs.emit(
     &mut log_record,
     BUILT_IN_INSTRUMENTATION_SCOPE.get().unwrap(),
   );
 }
 
+fn owned_string<'s>(
+  scope: &mut v8::HandleScope<'s>,
+  string: v8::Local<'s, v8::String>,
+) -> String {
+  let x = v8::ValueView::new(scope, string);
+  match x.data() {
+    v8::ValueViewData::OneByte(bytes) => {
+      String::from_utf8_lossy(bytes).into_owned()
+    }
+    v8::ValueViewData::TwoByte(bytes) => String::from_utf16_lossy(bytes),
+  }
```

Comment:
Is there a reason to do it instead of `v8::String::to_rust_string_lossy()`?

Reply:
https://github.com/denoland/rusty_v8/pull/1645